src/bitcoin-chainstate.cpp

```diff
@@ -49,7 +51,7 @@ int main(int argc, char* argv[])
     SelectParams(CBaseChainParams::MAIN);
     const CChainParams& chainparams = Params();
 
-    init::SetGlobals(); // ECC_Start, etc.
+    kernel::Context context{};
```

Comment:
The variable of type `kernel::Context` is called `context` here and `kernel` inside `struct NodeContext`. `kernel_context`, `kernel_state` or `kernel_ctx` may be better.

Reply:
Fixed in 319031b9798f1ed8f0a06e679284be26e0b42c92

Reply:
Now it is called `kernel_context` here and `kernel` elsewhere. Not a big deal, but I think it is better to be consistent if there are no other reasons for the naming.